experiences/src/main/kotlin/io/rover/campaigns/experiences/ExperiencesAssembler.kt

```diff
@@ -33,6 +36,17 @@ class ExperiencesAssembler : Assembler {
                 resolver.resolveSingletonOrFail(Context::class.java)
             )
         }
+
+        container.register(
+            Scope.Singleton,
+            ContextProvider::class.java,
+            "conversions"
+        ) { resolver ->
+            ConversionsContextProvider(
+                null,
+                resolver.resolveSingletonOrFail(LocalStorage::class.java)
```

Comment:
[NP] really minor but probably no point adding null here in the constructor if it's always going to be null, unless you're doing it for testing or following the DIP. I think you maybe wanna either just have it as a default arg and swap the param order, or pass an event emitter on the start listening call

Reply:
Actually this really doesn't matter either way ^ only change if you want